src/logic/tokens/store/actions/loadActiveTokens.js

```diff
@@ -12,7 +12,9 @@ const loadActiveTokens = () => async (dispatch: ReduxDispatch<GlobalState>) => {
   try {
     const tokens: Map<string, TokenProps> = await getActiveTokens()
     const tokenRecordsList: List<Token> = List(
-      Object.values(tokens).map((token: TokenProps): Token => makeToken(token)),
+      Object.values(tokens)
+        .filter((t) => typeof t.decimals !== 'string')
```

Comment:
could you maybe add a comment there explaining why we're doing the filter (link to your PR comment with explanation would be enough)

Reply:
Yeah good idea, we will forget about this in the future, will add it